projects/plugins/jetpack/extensions/blocks/conversation/edit.js

```diff
@@ -21,9 +23,9 @@ import { list as defaultParticipants } from './participants.json';
 
 const TRANSCRIPTION_TEMPLATE = [
 	[ 'core/heading', { placeholder: __( 'Conversation title', 'jetpack' ) } ],
-	[ 'jetpack/dialogue', defaultParticipants[ 0 ] ],
-	[ 'jetpack/dialogue', defaultParticipants[ 1 ] ],
-	[ 'jetpack/dialogue', defaultParticipants[ 2 ] ],
+	[ 'jetpack/dialogue', { participant: defaultParticipants[ 0 ] } ],
+	[ 'jetpack/dialogue', { participant: defaultParticipants[ 1 ] } ],
+	[ 'jetpack/dialogue', { participant: defaultParticipants[ 2 ] } ],
```

Comment:
New attribute shape for dialogue block: `participant` object with all properties (`label`, `slug`, etc), rather than storing only the `participantSlug`